IPython/core/display.py

```diff
@@ -475,8 +475,13 @@ class Latex(TextDisplayObject):
 
     def _repr_latex_(self):
         return self.data
+        
+        
+class MathML(TextDisplayObject):
```

Comment:
I don't see an obvious reason why `Math` shouldn't also support MathML. It would  have to have an extra switch, e.g. `Math('...', format='mathml')`. But I'm also not especially attached to doing it that way, I just think it's worth considering.